utopia-remix/app/test-util.ts

```diff
@@ -87,6 +101,17 @@ export async function truncateTables(models: DeletableModel[]) {
   }
 }
 
+export async function clearDb(prisma: UtopiaPrismaClient) {
```

Comment:
I actually added it to have a place where you know it "clears everything", instead of having your tests fail in the future just because someone added a new table that is not being cleared with the explicit `truncateTables([...])` call...
(so for example if we're adding a new `ProjectAccessRequest` table, we only need to make sure to clear it here instead of having to chase all the tests that are calling `truncateTable` and changing them)
wdyt? don't you think it's more robust to have one place that is in charge of "resetting" the db for tests?

Reply:
I mean, the "really" clean way of doing a reset would be to `truncate` the db and apply the migrations to start from a clean slate.
My point is: do you want a single function that clears everything? Cool, but then let's use it _everywhere_. The current solution/convention applied consistently in all test files is to be very explicit and deliberate about the db set up and teardown.
The motivations for a function to which you explicitly pass a list of tables to clean up are multiple, it forces you to be deliberate and think about the tables that are involved in the things you're testing (e.g. including cascades) and it scales well.

However a single cleanup function option that zaps everything can be fine too (e.g. in the future we need to seed data globally for the tests, in that case it can get handier). I picked my option when writing the initial tests but I'm fine with changing it, but please let's pick an option and be consistent with it.

My suggestion would be to not inflate this already huge diff with this kind of refactors and have a separate PR for them instead.

Reply:
good point about keeping this PR focused - changed it back to `truncateTables` 👍